### ISSUE TITLE
react-tools deprecated

### ISSUE THREAD

**Issue by eriknyk** [jenseng/react-i18nliner#19]
Hi, doing npm install for the project I'm getting this warning:

```
(master) % cd react-i18nliner                                                                                                              ~/towbook
(master) % npm install                                                                                            ~/towbook/react-i18nliner
npm WARN package.json react-i18nliner@0.0.12 No repository field.
npm WARN engine jest-cli@0.2.2: wanted: {"node":"0.8.x || 0.10.x"} (current: {"node":"0.12.4","npm":"2.10.1"})
npm WARN deprecated react-tools@0.12.2: react-tools is deprecated. For more information, visit https://fb.me/react-tools-deprecated
npm WARN prefer global react-tools@0.13.3 should be installed with -g
```

**Comment by jenseng** [comment]
Thanks for the bug report, I'll fix this in the next version :+1:

**Comment by jenseng** [comment]
whoops, this got fixed a while ago in v0.1.0
